lectures/dynamic_programming/mccall_model.md

```diff
@@ -427,7 +427,7 @@ This coding pattern, where `expression || error("failure)` first checks the expr
 Let's compute the reservation wage at the default parameters
 
 ```{code-cell} julia
-mcm = @with_kw (c=25.0, β=0.99, w=w) # named tuples
+mcm(c = 25.0, beta = 0.99, w) = (; c, beta, w)# named tuples
```

Comment:
Are you sure this is working?  Normally you require for all arguments after your first one.  This doesn't smell right.

Reply:
Also, since it doesn't have `;` this wouldn't be creating a kwarg one.  So below when you call `mcm(;c, beta)` it couldn't be using this funciotn.